tests/dummy/app/templates/form.hbs

```diff
@@ -3,4 +3,33 @@
 <TestForm
   @form={{@model.form}}
   @formStore={{@model.formStore}}
-/>
\ No newline at end of file
+/>
+
+
+<AuHeading
+  @level={{1}}
+  @skin={{3}}
+>
+  Data
+</AuHeading>
+<AuTable
+  @size="large"
+>
+  <:title>Test</:title>
+  <:header>
+  <tr>
+    <th>Subject</th>
+    <th>Predicate</th>
+    <th>Object</th>
+  </tr>
+  </:header>
+  <:body>
+  {{#each this.datasetTriples as |triple|}}
```

Comment:
Nice addition! although it does get very long on some forms, so we can probably improve the rendering in the future.